modules/ROOT/pages/branded-emails.adoc

```diff
@@ -0,0 +1,82 @@
+. Enter this code block:
++
+----
+    trigger AnypointEmailNotificationsTrigger on acm_pkg__AnypointContracts__ChangeEvent (after insert) {
+        List<acm_pkg__AnypointContracts__ChangeEvent> events = Trigger.new;
+        acm_pkg__.AnypointEmailNotifications.notifyOnContractUpdates(events);
+    }
+----
++
+. Navigate to *Setup* and use *Quick Find* to go to *Organization-Wide Addresses*.
+. Set *Display Name* to `EventEmailSender`.
+. Set *Email Address* to the email address at your organization that sends the API contract event notification emails.
++
+Ensure you can read confirmation emails sent to this address.
+. Set *Profiles* to `System Administrator`.
+. Click *Save*.
+. Open the confirmation email sent to the notification sender email address and click the link to confirm the address.
+. Navigate to *Setup* and use *Quick Find* to go to *Custom Metadata Types*.
+. Click *ACM Community Config Metadata* to add a record.
+. Set *Label* to your API community's name.
+. Set *ACM Community Config Metadata Name* to your API community's name.
+. Open the *Developer Console* and open the *Query Editor*.
+. Enter `SELECT Id, Name FROM Network` and click *Execute*.
+. In the list of IDs and communities, choose the row with *Name* equal to your community's name, double click the ID, and copy it.
+. In the *ACM Community Config Metadata* field *NetworkId*, paste the ID.
+. In the *Query Editor*, enter `SELECT Id, Name FROM EmailTemplate` and click *Execute*.
+. In the list of IDs and email templates, choose the row with *Name* equal to *API Community Manager: Contract Approved*, double click the ID, and copy it.
+. In the *ACM Community Config Metadata* field *ContractApprovedEmailTemplateId*, paste the ID.
+. In the list of IDs and email templates, choose the row with *Name* equal to *API Community Manager: Contract Revoked*, double click the ID, and copy it.
+. In the *ACM Community Config Metadata* field *ContractRevokedEmailTemplateId*, paste the ID.
+. In the *Query Editor*, enter `SELECT Id, Address FROM OrgWideEmailAddress` and click *Execute*.
+. In the list of IDs and email addresses, choose the row with *Address* equal to the organization wide email address you set, double click the ID, and copy it.
+. In the *ACM Community Config Metadata* field *OrgWideEmailAddressId*, paste the ID.
+. Click *Save*.
+. Repeat this process for all additional communities, creating one record for each community.
++
+*NetworkId* is different for each community. *ContractApprovedEmailTemplateId*, *ContractRevokedEmailTemplateId*, and *OrgWideEmailAddressId* may be the same for multiple communities.
+. Navigate to *Setup*, use *Quick Find* to go to *Classic Email Templates*, and navigate to *Folder* > *ACM*.
+. Customize the *API Community Manager: Contract Approved* and *API Community Manager: Contract Revoked* templates, or create your own templates and configure the *ACM Community Config Metadata* to use them.
```

Comment:
Names templates are: **Contract Approved** and **Contract Revoked**. (The first part of the name, API Community Manager: was removed because you are inside ACM Folder and it is redundant)

Reply:
@aamura thank you! Please review the fix in https://github.com/mulesoft/docs-api-community-manager/pull/141